features/admin.logs.v1/pages/logs-page.tsx

```diff
@@ -72,62 +71,130 @@ const LogsPage: FunctionComponent<LogsPageInterface> = (
+                data-componentid="log-publish-configuration"
+            >
+                <List>
+                    <List.Item>
+                        <Grid verticalAlign="middle">
+                            <Grid.Column
+                                floated="left"
+                                mobile={ 16 }
+                                computer={ 8 }
+                                verticalAlign="middle"
+                            >
+                                <List.Content verticalAlign="middle">
+                                    <GenericIcon
+                                        icon={ getMiscellaneousIcons().tenantIcon }
+                                        floated="left"
+                                        size="mini"
+                                        spaced="right"
+                                        verticalAlign="middle"
+                                        inline
+                                        square
+                                        transparent
+                                    />
+                                    <List.Header
+                                        data-componentid="application-consumer-account-link-title"
+                                        className="logs-title"
+                                    >
+                                        { t("console:manage.features.serverConfigs.remoteLogPublishing.title") }
+                                    </List.Header>
+                                    <List.Description
+                                        data-componentid="application-consumer-account-link-description"
+                                    >
+                                        { t("console:manage.features.serverConfigs.remoteLogPublishing.description") }
+                                    </List.Description>
+                                </List.Content>
+                            </Grid.Column>
+                            <Grid.Column
+                                mobile={ 16 }
+                                computer={ 4 }
+                                textAlign={ "right" }
+                            >
+                                { (
+                                    <Popup
+                                        trigger={ (
+                                            <Button
+                                                className="log-publish-settings-button"
+                                                data-componentid="navigate-to-log-publish-settings-button"
+                                                onClick={ handleSettingsButton }
+                                            >
+                                                <GearIcon /> Configure
+                                            </Button>
+                                        ) }
+                                        content={ "Remote Log Publish Settings" }
```

Comment:
Let's i18n this string literal.

```suggestion
                                        content="Remote Log Publish Settings"
```